protocol/messenger_mailserver_cycle.go

```diff
@@ -172,7 +172,7 @@ func (m *Messenger) getFleet() (string, error) {
 	} else if m.config.clusterConfig.Fleet != "" {
 		fleet = m.config.clusterConfig.Fleet
 	} else {
-		fleet = params.FleetProd
+		fleet = params.FleetShardsTest
```

Comment:
I have moved DefaultFleet to the params package.

Otherwise, if I use `api.DefaultFleet` here, then there is a cyclyc dependency:

```
package github.com/status-im/status-go/build/bin/statusgo-lib
	imports github.com/status-im/status-go/mobile
	imports github.com/status-im/status-go/api
	imports github.com/status-im/status-go/node
	imports github.com/status-im/status-go/services/accounts
	imports github.com/status-im/status-go/protocol
	imports github.com/status-im/status-go/api: import cycle not allowed
```